docs/install.md

```diff
@@ -152,7 +168,7 @@ yarn global add code-server
 # Or: npm install -g code-server
 code-server
 # Now visit http://127.0.0.1:8080. Your password is in ~/.config/code-server/config.yaml
-```
+````
```

Comment:
There's an extra backquote here that I think maybe shouldn't be here? - surprised it went through `yarn fmt` without complaining actually :eyes:

Reply:
whoops!